exporter/azure_monitor/utils/utils.go

```diff
@@ -48,3 +50,12 @@ func FormatTime(t time.Time) string {
 	formattedTime := t.Format(TimeFormat)
 	return formattedTime
 }
+
+/* Get netloc of url given a string */
+func UrlToDependencyName(inputUrl string) string {
+	urlParsed, err := url.ParseRequestURI(inputUrl)
+	if err != nil {

```

Comment:
If this fails, is the error handled properly when this function is called in azure_monitor.go?

Reply:
No, but it should be. Nice catch.

Reply:
Still need to fix this.